packages/jsdoc-plugins/lib/doclet.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Doclets are atomic parts of the JSDoc output that describe each symbol of parsed input.
+ *
+ * @typedef {Object} Doclet
+ * @property {String} longname Long name of the doclet (including the `module:` part).
+ * @property {String} name Short name of the doclet (e.g. the name of the method).
+ * @property {String} memberof Where the doclet belongs to (parent of the symbol).
+ * @property {'class'|'interface'|'mixin'|'function'|'typedef'|'event'|'member'|'constant'|'module'} kind The kind of the doclet's symbol.
+ * @property {Boolean} [ignore] `true` for internal doclets which should not be published
+ * @property {Boolean} [undocumented] `true` when a doclet's symbol does not have API docs written above the declaration.
+ * @property {String} [inheritdoc] Warning: When the `@inheritdoc` is present, then in most cases the property
+ * becomes  an empty string!
```

Comment:
Why does it become to be an empty string? It should be explained. I was reading it a few times and I don't get it.

Reply:
Because when JSDoc sees the `@inheritDoc` tag, then this property becomes `""`. Otherwise, it becomes undefined. Most probably due to the fact that after the `@inheritDoc` tag you should be able to type more docs for that specific symbol.

Reply:
When I wrote: "It should be explained.", I meant: "an answer for the asked question could expand the description ".